wrangler-core/src/main/java/io/cdap/directives/column/CleanseColumnNames.java

```diff
@@ -93,4 +102,30 @@ public Mutation lineage() {
       .all(Many.of())
       .build();
   }
+
+  @Override
+  public Relation transform(RelationalTranformContext relationalTranformContext,
+                            Relation relation) {
+    java.util.Optional<ExpressionFactory<String>> expressionFactory = SqlExpressionGenerator
+            .getExpressionFactory(relationalTranformContext);
+    if (!expressionFactory.isPresent()) {
+      return new InvalidRelation("Cannot find an Expression Factory");
+    }
+    List<String> columnNames = SqlExpressionGenerator.generateListCols(relationalTranformContext);
```

Comment:
So this method gets the list of input columns from the transform context, which is not really related to "generating a SQL expression". I would either suggest keeping this in a different util class or you can just rename the method to getColumnList